pkgs/development/tools/jless/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jless";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "PaulJuliusMartinez";
+    repo = "jless";
+    rev = "v${version}";
+    sha256 = "sha256-gBqyo/N/qF6HCTUrSKNVLiL1fc/JTfip1kNpNCBzRT8=";
+  };
+
+  cargoSha256 = "sha256-PbX61RVbrI2kTuyXK+LhQdJDvNo3KjIQH5eBbL6iUBM=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A command-line pager for JSON data.";
```

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

`meta.description` shouldn't end with a period.

```suggestion
    description = "A command-line pager for JSON data";
```

---

Fixed!